src/main/java/com/zufar/onlinestore/payment/enums/PaymentStatus.java

```diff
@@ -1,12 +1,37 @@
 package com.zufar.onlinestore.payment.enums;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
 public enum PaymentStatus {
-    REQUIRES_PAYMENT_METHOD,
-    REQUIRES_CONFIRMATION,
-    REQUIRES_CAPTURE,
-    REQUIRES_ACTION,
-    PROCESSING,
-    CANCELED,
-    SUCCEEDED
+
+    PAYMENT_METHOD_IS_REQUIRED("payment_intent.requires_payment_method",
```

Comment:
Why status and message are not equal?
it is written that status is `payment_intent.requires_payment_method`, but message is `Payment method is required`.
Why status is not  `payment_intent.payment_method is required`?
if you agree, fix it everywhere , please

Reply:
> payment_intent.requires_payment_method

payment_intent.requires_payment_method it is constant wich managed by stripe